addons/l10n_it_edi_extended/models/account_chart_template.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = 'account.chart.template'
+
+    def _load(self, sale_tax_rate, purchase_tax_rate, company):
```

Comment:
maybe this can go in l10n_it_edi anyways?  (not sure why 22% tax is sometimes missing in demo data)

Reply:
22% tax is "new" from 2013.
It could go on `l10n_it_edi`, true.